website/events/api/calendarjs/serializers.py

```diff
@@ -26,6 +31,32 @@ def _class_names(self, instance):
             # I think this handles the case that registration is needed, but not yet possible
             return ["regular-event-registration-closed"]
 
+    def _registration_info(self, instance: Event):
+        if self.context["member"] and services.is_user_registered(
+            self.context["member"], instance
+        ):
+            queue_pos = services.user_registration_pending(
+                self.context["member"], instance
+            )
+            if isinstance(queue_pos, int):
+                return f"In waiting list at position {queue_pos}"
```

Comment:
Can these strings get put into `_` for localisation, just to keep it consistent.